canonical-web-and-design/snapcraft.io
# Display truncated fsf yaml files

## Done

- Display truncated yaml files on FSF

## Issue / Card

Fixes #2267 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/first-snap
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click on any language and see the yaml files are truncated and you can `Show more` by clicking on the link
- Choosing a different language should reset the previously expanded yaml file to being truncated
